config/application.rb

```diff
@@ -46,7 +43,7 @@ class Application < Rails::Application
 
     # Don't generate system test files.
     config.generators.system_tests = nil
-    config.eager_load_paths << Rails.root.join("lib")
+    config.eager_load_paths << Rails.root.join("lib").to_s
```

Comment:
Weird, searching the internet suggests this is because `Rails.root` is a `Pathname`, I guess it wasn't working before adding `.to_s`.

Reply:
I thought it wasn't but after reading your comment I tried again and it works without the .to_s as well so I removed it again